analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -61,6 +61,17 @@ public class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
 
     protected override void Initialize(SonarAnalysisContext context)
     {
+        context.RegisterNodeAction(
+        c =>
+        {
+            var compilation = (CompilationUnitSyntax)c.Node;
```

Comment:
Because `compilation` refers to `Compilation` type in the code base
```suggestion
            var compilationUnit = (CompilationUnitSyntax)c.Node;
```